system/mathLoadTest/playlist.xml

```diff
@@ -129,6 +129,10 @@
 				<comment>rtc 139897</comment>
 				<variation>Mode554</variation>
 			</disable>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/14590</comment>
+				<platform>aarch64_mac.*</platform>
```

Comment:
We can use `<platform>aarch64_mac</platform>` without `.*`

Reply:
No, I need `.*` here because we only have `aarch64_mac_xl` build at the moment.